samples/java-customer-registry/src/main/java/customer/CustomerValueEntity.java

```diff
@@ -19,39 +19,39 @@
 import com.akkaserverless.javasdk.valueentity.CommandContext;
 import com.akkaserverless.javasdk.valueentity.CommandHandler;
 import com.akkaserverless.javasdk.valueentity.ValueEntity;
+import com.akkaserverless.javasdk.valueentity.ValueEntityBase;
 import com.google.protobuf.Empty;
 import customer.api.CustomerApi;
 import customer.domain.CustomerDomain;
 
 @ValueEntity(entityType = "customers")
-public class CustomerValueEntity {
+public class CustomerValueEntity extends ValueEntityBase<CustomerDomain.CustomerState> {
```

Comment:
Side question: will we be renaming `ValueEntityBase` to just `ValueEntity` once we drop the annotations?

Reply:
that was my intention for both ValueEntityBase and EventSourcedEntityBase (I think there is a fixme somewhere)